cognite_toolkit/_builtin_modules/inrobot/cdf_inrobot_common/module.toml

```diff
@@ -1,7 +1,11 @@
 [module]
 title = "InRobot Common"
+default_selected = true
 
 [packages]
 tags = [
     "inrobot",
 ]
+
+[dependencies]
+modules = ["cdf_apm_base"]
```

Comment:
Very explicit, but I think that is good.

Just to show an alternative, which I don't thin is necessarily better.

```yaml
[module]
...
dependencies = ["cdf_apm_base"]

```